rocAL/rocAL/source/rali_api_data_loaders.cpp

```diff
@@ -740,6 +740,160 @@ raliJpegCaffeLMDBRecordSourceSingleShard(
     return output;
 }
 
+RaliImage  RALI_API_CALL
+raliMXNetRecordSource(
+        RaliContext p_context,
+        const char* source_path,
+        RaliImageColor rali_color_format,
+        unsigned internal_shard_count,
+        bool is_output,
+        bool shuffle,
+        bool loop,
+        RaliImageSizeEvaluationPolicy decode_size_policy,
+        unsigned max_width,
+        unsigned max_height)
+{
+    Image* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
```

Comment:
please add the following code for checking null context. Applicable to all new APIs.
    if (p_context == nullptr) {
        ERR("Invalid RALI context or invalid input image")
        return output;
    }

Reply:
Added the codes